src/ai/backend/agent/kubernetes/intrinsic.py

```diff
@@ -133,9 +143,51 @@ async def extra_info(self) -> Mapping[str, str]:
         }
 
     async def gather_node_measures(self, ctx: StatContext) -> Sequence[NodeMeasurement]:
-        # TODO: Create our own k8s metric collector
+        async def get_cpu_stat(core_api, name: str):
+            try:
+                raw_resp = await core_api.connect_get_node_proxy_with_path(
+                    name=name, path="metrics/resource"
+                )
+            except K8sApiException as e:
+                log.warning("kubernetes api error: {}", e)
+                return None
+            resp = [line for line in raw_resp.split("\n") if "#" not in line and "node_cpu" in line]
+            cpu_used = resp.pop().split(" ")[1]
+            return cpu_used
 
-        return []
+        await K8sConfig.load_kube_config()
```

Comment:
Is it inevitable to load the configuration here? Could we do it somewhere else/earlier and avoid repeating this on every stats measurement?